dev/shim/dune

```diff
@@ -2,6 +2,7 @@
  (targets coqtop-prelude)
  (deps
   %{bin:coqtop}
+   ; XXX: bug, we are missing the dep on the _install meta file...
```

Comment:
Is it possible to directly add the dep on the META?

Reply:
Not sure, the build vs install layout is biting us here maybe.

Reply:
I think we will keep using the legacy method for now, and we can address moving to the META in a different PR. It is not very urgent as we need to keep the legacy method until we fix dune.

We can add (if its not there yet) `%{pkg:coq-core:META}` as a dune variable, that is the same tho than expanding it and depending on the install location. Maybe we can do better then and add the rules for a `_build/.lib` layout, expanding OCAMLPATH.

Anyways, both concerns for future PR.

Reply:
So to reply to my less informed past self, `%{lib:coq-core:META}` does not work and atm dune can only rely on `(package coq-core)` to build that META.